test/vm_tests/builder/codegen/names_test.dart

```diff
@@ -37,6 +40,9 @@ main() {
 
         test('consumerName', () => expect(names.consumerName, r'foo.Foo'));
         test('implName', () => expect(names.implName, r'foo._$Foo'));
+        test('privateConfigName', () => expect(names.privateConfigName, r'_$foo.FooConfig'));
+        // ignore: deprecated_member_use_from_same_package
+        test('publicConfigName', () => expect(names.publicConfigName, r'$foo.FooConfig'));
```

Comment:
These should actually be:
```suggestion
        test('privateConfigName', () => expect(names.privateConfigName, r'foo._$FooConfig'));
        // ignore: deprecated_member_use_from_same_package
        test('publicConfigName', () => expect(names.publicConfigName, r'foo.$FooConfig'));
```
My bad for messing up configName and not testing it 😅 

You can look at `implName` to see how they should be implemented